### ISSUE TITLE
Device Error Received, code3 

### ISSUE THREAD

**Issue by shreks7** [logits-systems/Cam2Ndk#4]
I am getting errors on Samsung Galaxy S8

```
[ImageReader-1280x720f23u3m1-6587-0](id:19bb00000002,api:4,p:1713,c:6587) 
dequeueBuffer: waitForFreeSlotThenRelock: timeout. dequeuedCount: 0, acquiredCount: 0
Device error received, code 3, frame number 10, request ID 0, subseq ID 0
```

Any ideas?

**Comment by shreks7** [comment]
My bad, I wasn't doing anything in the `Image Processing Loop`  and not calling `AImage_delete(yuv_image);`
